docs/applications/rubintv/index.rst

```diff
@@ -0,0 +1,23 @@
+.. px-app:: rubintv
+
+##########################
+rubintv
+##########################
+
+rubintv is the display front end for ... well, stuff.  There are also
+some worker pods supporting that front-end pod.  Merlin Fisher-Levine
+understands it, but the rest of us really don't.
```

Comment:
I think merlin provided an updated introduction. Also, the title of every application must be `name — short description`